deployment_scripts/appimage/build_appimage.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+python_cmd="python3.7"
+APPDIR="$HOME/monkey-appdir"
+INSTALL_DIR="$APPDIR/usr/src"
+
+GIT=$HOME/git
+
+REPO_MONKEY_HOME=$GIT/monkey
+REPO_MONKEY_SRC=$REPO_MONKEY_HOME/monkey
+
+ISLAND_PATH="$INSTALL_DIR/monkey_island"
+MONGO_PATH="$ISLAND_PATH/bin/mongodb"
+ISLAND_BINARIES_PATH="$ISLAND_PATH/cc/binaries"
+
+is_root() {
+  return $(id -u)
+}
+
+has_sudo() {
+  # 0 true, 1 false
+  return $(sudo -nv > /dev/null 2>&1)
+}
+
+handle_error() {
+  echo "Fix the errors above and rerun the script"
+  exit 1
+}
+
+log_message() {
+  echo -e "\n\n"
+  echo -e "DEPLOYMENT SCRIPT: $1"
+}
+
+setup_appdir() {
+    rm -rf $APPDIR | true
+    mkdir -p $INSTALL_DIR
+}
+
+install_pip_37() {
+    pip_url=https://bootstrap.pypa.io/get-pip.py
+    curl $pip_url -o get-pip.py
+    ${python_cmd} get-pip.py
+    rm get-pip.py
+}
+
+install_nodejs() {
+  log_message "Installing nodejs"
+  node_src=https://deb.nodesource.com/setup_12.x
+  curl -sL $node_src | sudo -E bash -
+  sudo apt-get install -y nodejs
+}
+
+install_build_prereqs() {
+    # appimage-builder prereqs
+    sudo apt install -y python3 python3-pip python3-setuptools patchelf desktop-file-utils libgdk-pixbuf2.0-dev fakeroot strace
+
+    #monkey island prereqs
+    sudo apt install -y curl libcurl4 python3.7 python3.7-dev openssl git build-essential moreutils
+    install_pip_37
+    install_nodejs
+}
+
+install_appimage_builder() {
+    sudo pip3 install appimage-builder
+
+    install_appimage_tool
+}
+
+install_appimage_tool() {
+    APP_TOOL_BIN=$HOME/bin/appimagetool
+    mkdir $HOME/bin
+    curl -L -o $APP_TOOL_BIN https://github.com/AppImage/AppImageKit/releases/download/12/appimagetool-x86_64.AppImage
+    chmod u+x $APP_TOOL_BIN
+
+    PATH=$PATH:$HOME/bin
+}
+
+load_monkey_binary_config() {
+  tmpfile=$(mktemp)
+
+  log_message "downloading configuration"
+  curl -L -s -o $tmpfile "$config_url"
+
+  log_message "loading configuration"
+  source $tmpfile
+}
+
+clone_monkey_repo() {
+  if [[ ! -d ${GIT} ]]; then
+    mkdir -p "${GIT}"
+  fi
+
+  log_message "Cloning files from git"
+  branch=${2:-"local-run-data-dir"}
```

Comment:
will get re-targeted in a different PR? Make a const?

Reply:
I'll just switch it to develop for now.